src/components/Header/HeaderContact.js

```diff
@@ -46,7 +46,7 @@ const HeaderContact = ({
   return (
     <View style={[styles.container, containerStyle]}>
       {renderHeader()}
-      <Text style={[styles.text, titleStyle]}>{title}</Text>
+      <Text style={[styles.text, styles.titleText, titleStyle]}>{title}</Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a new style `titleText` to the Text component is fine as long as it doesn't conflict with the existing styles and meets the design requirements.
```diff
- <Text style={[styles.text, titleStyle]}>{title}</Text>
+ <Text style={[styles.text, styles.titleText, titleStyle]}>{title}</Text>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->